OpenSim/Tests/ControllerExample/testControllerExample.cpp

```diff
@@ -32,10 +32,16 @@
 using namespace OpenSim;
 using namespace std;
 
+void changeVersionNumber(const std::string& filenameOld,
+                         const std::string& filenameNew);
+
 int main()
 {
     try {
-        Storage result1("tugOfWar_controls.sto"), 
+      const std::string result1Filename{"tugOfWar_controls.sto"};
+      const std::string result1FilenameV2{"tugOfWar_controls_v2.sto"};
+      changeVersionNumber(result1Filename, result1FilenameV2);
```

Comment:
The naming makes this a bit hard to follow. One would expect  `result1FilenameV2` to be v2 or an update in versioning but you are in fact downgrading the version from 2 to 1.